src/bridge/http.js

```diff
@@ -33,9 +33,16 @@ function contentType(body: any): string {
   }
 }
 
+function cloneString(data: string) {
+  // Electron v23.x crashes when using fetch + Buffer.toString('hex')
+  // temp solution is to clone the string value.
+  // https://github.com/electron/electron/issues/38295
+  return JSON.parse(JSON.stringify(data))
```

Comment:
Is the purpose if this function to clone object or string? @kewde

Reply:
@thanhdanh string but it can also clone objects